src/ui/public/doc_table/components/table_header.js

```diff
@@ -53,13 +53,13 @@ module.directive('kbnTableHeader', function (shortDotsFilter) {
       };
 
       $scope.headerClass = function (column) {
-        if (!isSortableColumn(column)) return;
+        if (!isSortableColumn(column)) return ['docTableHeaderButton'];
 
         const sortOrder = $scope.sortOrder;
-        const defaultClass = ['fa', 'fa-sort-up', 'table-header-sortchange'];
+        const defaultClass = ['fa', 'fa-sort-up', 'table-header-sortchange', 'docTableHeaderButton'];
 
         if (!sortOrder || column !== sortOrder[0]) return defaultClass;
-        return ['fa', sortOrder[1] === 'asc' ? 'fa-sort-up' : 'fa-sort-down'];
+        return ['fa', 'docTableHeaderButton', sortOrder[1] === 'asc' ? 'fa-sort-up' : 'fa-sort-down'];
```

Comment:
Because `docTableHeaderButton` is going to be applied in every case, we can actually remove it from this logic (and go back to what was here originally), and use it directly in the template instead by assigning it with the `class` attribute:

```html
    <span>Time <button
      id="docTableHeaderFieldSort{{indexPattern.timeFieldName}}"
      tabindex="0"
      aria-label="{{ getAriaLabelForColumn(indexPattern.timeFieldName) }}"
      class="docTableHeaderButton"
      ng-class="headerClass(indexPattern.timeFieldName)"
      role="button"
      ng-click="cycleSortOrder(indexPattern.timeFieldName)"
      tooltip="Sort by time"
    ></button>
    </span>

      <button
        id="docTableHeaderFieldSort{{name}}"
        tabindex="0"
        aria-label="{{ getAriaLabelForColumn(name) }}"
        class="docTableHeaderButton"
        ng-class="headerClass(name)"
        ng-click="cycleSortOrder(name)"
        tooltip="{{tooltip(name)}}"
        tooltip-append-to-body="1"
      ></button>
```